packages/twenty-server/src/workspace/messaging/services/messaging-utils.service.ts

```diff
@@ -202,39 +202,39 @@ export class MessagingUtilsService {
     );
   }
 
-  public async getConnectedAccountsFromWorkspaceId(
-    workspaceId: string,
+  public async getConnectedAccounts(
```

Comment:
Note for next PR: getConnectedAccounts and getConnectedAcountByIdOrFail should be moved to a dedicated service